ci-operator/step-registry/lp-interop-tooling/create-s3/lp-interop-tooling-create-s3-commands.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
```

Comment:
If we create this we are signing up to maintain an s3 bucket creation script for everyone that uses OpenShift CI. Is this something that our team should be doing?

Does the OADP repos have a script that they maintain that creates s3 buckets? If they do, they are already maintaining that code and we should create a step to use their code which is being maintained already, not creating something here that we need to maintain.

Reply:
There is a script in the oadp-qe-automation repo for deploying an S3 bucket.  https://github.com/CSPI-QE/oadp-qe-automation/blob/main/backup-locations/aws-s3/deploy.sh